apps/azure-devops/azure-devops-agent/image-repo.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: image.toolkit.fluxcd.io/v1alpha2
 kind: ImageRepository
 metadata:
-  name: az-devops-agent
+  name: azure-devops-agent-test
```

Comment:
why does this have test in the name?

Reply:
doh, was testing it locally and forgot to remove it